test/e2e/opaque_resource.go

```diff
@@ -38,7 +38,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = framework.KubeDescribe("Opaque resources [Feature:OpaqueResources]", func() {
+var _ = framework.KubeDescribe("Opaque resources [Serial]", func() {
```

Comment:
It was not serial, then @timothysc suggested making it serial. I'm afraid I don't understand the implications enough to make an educated decision. Does it control parallelism of just this group of tests or does it span the entire e2e framework?